packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -166,7 +166,7 @@
                                 data-ng-repeat="node in $ctrl.database.nodes track by node.id"
                             >
                                 <p
-                                    data-ng-if="$ctrl.database.nodes.length > 4 && $index === $ctrl.ceil($ctrl.database.nodes.length / 2)"
+                                    data-ng-if="$ctrl.database.nodes.length > 4 && $index % 4 === 0"
```

Comment:
Please define the number as a constant to help understand the logic better

Reply:
Done: https://github.com/ovh/manager/pull/5431/commits/0acdeaf4481660a0d592e8516868d4794c6f2043